modules/ROOT/pages/map-users-roles-ldap-task.adoc

```diff
@@ -46,10 +49,36 @@ Mapping:
 +
 image::external-identity-cfb1e.png[]
 
-. Click Set names.
-.. To map more than one attribute name to the selected role, click Add More and add another attribute.
+. Click *Set names*.
++
+To map more than one attribute name to the selected role, click *Add More*, and add another attribute.
+
 . Repeat this process for each role that you want to map to an external group.
 
+== Configure Multiple Group Mappings for Single Sign-On
+
+Organizations that use single sign-on and require their users to obtain permissions from multiple groups can configure group mappings using their identity provider. If your IDP provides each group separately, you can create a group attribute that contains multiple groups you want to use. In many cases, you can create an array of groups.
+
+For example, your IDP may provide your groups in the following format:
+
+----
+<ns2:Attribute Name="Groups" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified">
+  <ns2:AttributeValue>Mule_Org_Admin_XXX</ns2:AttributeValue>
+</ns2:Attribute>
+<ns2:Attribute Name="Groups" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified">
+  <ns2:AttributeValue>Mule_Exchange_XXX</ns2:AttributeValue>
+</ns2:Attribute>
+----
+
+To prevent security vulnerabilities, Anypoint Platform requires you to create a single group that encompasses multiple group mappings. For example:
```

Comment:
```suggestion
To prevent security vulnerabilities, Anypoint Platform requires you to create a single group that encompasses multiple group mappings:
```
But should we have introduced the motivation of preventing security vulnerabilities in the topic intro?

Reply:
The vulnerabilities drive our requirement for the array, not their multiple group mappings. They have to make the array because *we* think they shouldn't have separate groups with commas between. As such, we do not allow them to add their groups to our field in that format.  

Their IDP will often present groups separately (as this is more efficient on the IDP's end), but I don't want to insinuate that the IDP's presentation is a problem, as I don't think it is.